tidb-cloud/airbyte.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect Airbyte with TiDB
+summary: Learn how to use Airbyte TiDB connector.
+---
+
+# Connect Airbyte with TiDB
+
+[Airbyte](https://airbyte.com/) is an open-source data integration engine that helps you consolidate your data in your data warehouses, lakes, and databases. This document shows how to connect Airbyte to TiDB as a source or a destination.
+
+## Deploy Airbyte
```

Comment:
Can we use steps for this and the following sections?

Reply:
These steps are only a part of all operations. We should not give them a “step relationship”